float64 x_control
float64 theta_control